Index.html

```diff
@@ -186,5 +186,253 @@ <h2 class="hero-text-title">I'm a Full Stack Web Developer</h2>
       </div>
 
     </section>
+
+    <section class="recent-works" id="recent-works-id">
+
+      <h2 class="recent-works-title"><span>My Recent Works</span></h2>
+
+      <div class="main-work-box">
+        <div class="main-work-image"></div>
+        <div class="main-work-text-box">
+          <h3 class="main-work-text-title">Lifestyle-Articles Web Application</h3>
+          <p class="main-work-text-area">
+          Based on the website design of <a href="https://www.behance.net/gallery/14554909/liFEsTlye-Mobile-version" class="nelson-sakwa">Nelson Sakwa on Behance</a>. Build with Ruby on Rails, the project has the models, views and controllers of each of the resources User, Article, Vote and Category. The User has authentication and can perform any of the CRUD operations on the Articles and can create and delete a Vote. Article and Category has a many to many relationship. The Article with the most Vote will be displayed in the hero section of the main page.
+          </p>
```

Comment:
- [ ] You did a good on the styles but for this project, you are supposed to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) design as much as possible. Please use the correct font size and font-family for the project description and make sure it matches with the Design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/166122700-cbb832a1-c6e2-49e9-8aef-556dac4e23e3.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/166122712-5e697995-3443-45de-899a-71b2b70bf24d.png)